src/Flow.jl

```diff
@@ -59,6 +59,15 @@ lowerBoundary!(r,u,Φ,ν,i,j,N,::Val{true}) = @loop (
     Φ[I] = ϕuP(j,CIj(j,CI(I,i),N[j]-2),CI(I,i),u,ϕ(i,CI(I,j),u)) -ν*∂(j,CI(I,i),u); r[I,i] += Φ[I]) over I ∈ slice(N,2,j,2)
 upperBoundary!(r,u,Φ,ν,i,j,N,::Val{true}) = @loop r[I-δ(j,I),i] -= Φ[CIj(j,I,2)] over I ∈ slice(N,N[j],j,2)
 
+
+function applyBodyForce!(r::AbstractArray{T,Dv},t,g=(a,b)->0) where {T,Dv}
```

Comment:
If `T,Dv` are not used within the function and there are no other `applyBofyForce!` function for which multiple dispatch would be required, there is not need to specify the type for `r`.
```suggestion
function applyBodyForce!(r,t,g=(a,b)->0)
```